CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Fixed
+- Fix legacy volume seekbar not visible when set at the lowest
```

Comment:
"Vertical `VolumeSlider` knob in legacy skin was not visible when set to a low volume"